docs/manual/docs/api/q-search.md

```diff
@@ -0,0 +1,100 @@
+# Q Search {#q-search}
+
+!!! warning
+
+    Unavailable since version 4.0.0.
+    
+    There is no known sponsor or interested party for implementing Q-Search
+    Interested parties may contact the project team for guidance and to express their intent.
+
+The Q Search endpoint allows you to query the catalog programmatically. It is available in the local catalog at `http://localhost:8080/geonetwork/srv/eng/q` (otherwise substitute your catalog URL).
+
+## Upgrading from GeoNetwork 3.0 Guidance
+
+The Q Search endpoint is replaced by the Elasticsearch ``/srv/api/search/records/_search`` endpoint.
```

Comment:
@josegar74 it is unclear to me if you are restoring q-search? If so we can remove this heading.

Reply:
I will remove this section. Is their any other feed back?

Reply:
There are no plans to restore the  `q` service. It should be used the Elasticsearch search endpoint sending with `POST` the queries in Elasticsearch syntax.

Reply:
I have made that clear in the revised text.